python/kfserving/kfserving/storage.py

```diff
@@ -268,8 +268,9 @@ def _download_from_uri(uri, out_dir=None):
         with requests.get(uri, stream=True, headers=headers) as response:
             if response.status_code != 200:
                 raise RuntimeError("URI: %s returned a %s response code." % (uri, response.status_code))
+            zip_content_types = ('application/x-zip-compressed', 'application/zip', 'application/zip-compressed')
             if mimetype == 'application/zip' and not response.headers.get('Content-Type', '')\
-                    .startswith('application/zip'):
+                    .startswith(zip_content_types):
                 raise RuntimeError("URI: %s did not respond with \'Content-Type\': \'application/zip\'" % uri)
```

Comment:
Can you help fix the error message with the variable `zip_content_types`?

Reply:
Fixed